pytest_html/plugin.py

```diff
@@ -86,7 +88,7 @@ def data_uri(content, mime_type='text/plain', charset='utf-8'):
 
 class HTMLReport(object):
 
-    def __init__(self, logfile):
+    def __init__(self, logfile, **kwargs):
```

Comment:
@davehunt I think didn't understand very well, are you suggesting to use single arguments instead of kwargs?  Or what?